File: insertionSort/insertionSort.c
void insertion_Sort(int *array, int size)
{
    int temp;
    for(int i = 1; i < size; i++)
    {
        int j = i;
        temp = array[j];

        while(temp < array[j-1] && j > 0)
        {
            array[j] = array[j-1];
            j--;
        }
        array[j] = temp;
    }
}

File: insertionSort/insertionSort.py
def insertion_sort(array):
    for i in range(1, (len(array))):
        j = i - 1
        temp = array[i]

        while temp < array[j] and j >= 0:
            array[j+1] = array[j]
            j -= 1

        array[j+1] = temp